$NetBSD: patch-aa,v 1.1.1.1 2010/08/25 07:34:11 manu Exp $
Do not require libpuffs from src, as we may not have the sources
--- lib/libperfuse/Makefile.orig	2010-08-23 14:44:21.000000000 +0200
+++ lib/libperfuse/Makefile	2010-08-23 14:44:28.000000000 +0200
@@ -1,6 +1,6 @@
 LIB=            perfuse
-LIBDPLIBS+=     puffs	/usr/src/lib/libpuffs
+#LIBDPLIBS+=     puffs	/usr/src/lib/libpuffs
 
 
 PERFUSE_OPT_DEBUG_FLAGS=   -g -DPERFUSE_DEBUG
 
